visualization/app/codeCharta/ui/screenshotButton/screenshotButton.component.ts

```diff
@@ -41,6 +66,25 @@ export class ScreenshotButtonController {
 	}
 
 	private loadScript(link: HTMLAnchorElement, renderer: WebGLRenderer) {
+		this.buildScreenShotCanvas(renderer)
+
+		link.href = renderer.domElement.toDataURL()
+		renderer.setPixelRatio(1)
```

Comment:
We should add a comment, why setPixelRatio must be called.

Reply:
apparently we do not need setPixelRatio